spec/motion-spec/bacon_spec.rb

```diff
@@ -367,11 +367,11 @@ def check(ctx, name)
   end
 
   it 'should work with namespaced modules' do
-    check(Kernel.send(:describe, Motion::Spec::Context) {}, 'Motion::Spec::Context')
+    check(Kernel.send(:describe, MotionSpec::Context) {}, 'MotionSpec::Context')
   end
 
   it 'should work with multiple arguments' do
-    check(Kernel.send(:describe, Motion::Spec::Context, :empty) {}, 'Motion::Spec::Context empty')
+    check(Kernel.send(:describe, MotionSpec::Context, :empty) {}, 'MotionSpec::Context empty')
```

Comment:
Line is too long. [94/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.